standalone/settings.gradle

```diff
@@ -35,7 +37,5 @@ gradle.beforeProject { project ->
             // Use this repository when relying on snapshot versions of LabKey artifacts or requiring snapshot external dependencies
```

Comment:
I think this can now be just "Use this repository when relying on snapshot versions of LabKey artifacts". It's possible we'll server up snapshots for  some of the locally-hosted external dependencies, but not what we really like to do. No big deal, though. Main thing is the ordering.